# PROJET SEATTLE ENERGY BENCHMARKING
## Notebook 00 : initialisation de l'environnement et ingestion des données

---

### Identité du document
* **Statut :** Phase 1 (exploration & prototypage)
* **Date de création :** 26 Décembre 2025
* **Dernière mise à jour :** 29 Décembre 2025
* **Dépendances notebooks**: Aucune

### Description
Ce notebook établit les fondations du projet. Il assure la mise en place de l'infrastructure et la configuration des outils de gestion . L'objectif est de garantir une base reproductible avant d'entamer l'audit de qualité.

### Objectifs principaux
1. Initialiser le système de gestion de configuration via Hydra.
2. Déployer l'arborescence des répertoires du projet.
3. Récupérer et charger le jeu de données source (Immuable).
4. Établir un premier diagnostic structurel des données chargées.

### Dépendances critiques
* `hydra` : Gestion de la configuration.
* `src.utils` & `src.data` : Modules internes de support.

### LIVRABLES
1. Environnement technique initialisé (Hydra, Dossiers).
2. Dataset brut dans `data/raw/`.
3. Dictionnaire de données initial (`reports/data_dictionary.md`).
4. Note de synthèse structurelle.
5. Notebook propre

---

# Table des Matières
- [Section 0 : Importation des packages ](#section-1--configuration)
- [Section 1 : Configuration et initialisation ](#section-1--configuration)
- [Section 2 : Contextualisation & provenance](#section-2--contextualisation--provenance)
- [Section 3 : Chargement et inspection structurelle](#section-3--ingestion-et-premier-contact)
- [Section 4 : Typologie des variables](#section-4--typologie-et-cartographie-des-variables)
- [Section 5 : Focus sur la variable cible](#section-5--analyse-de-la-variable-cible)
- [Section 6 : Identification des redondances apparentes](#section-6--étude-des-redondances-et-unités)
- [Section 7 :Premières incohérences détectées](#section-7--détection-des-signaux-dalerte-immédiats)
- [Section 8 : Synthèse et génération du dictionnaire](#section-8--synthèse-et-génération-du-dictionnaire)
---

# Section 0 : Importation des packages

In [1]:
import logging
import pandas as pd
from pathlib import Path


# Import des fonctions utilitaires 

import sys
from pathlib import Path

PROJECT_ROOT = Path.cwd().parent
SRC_PATH = PROJECT_ROOT / "src"

if str(SRC_PATH) not in sys.path:
    sys.path.insert(0, str(SRC_PATH))


from data.load_data import load_data_raw
from utils.config_loader import load_config, create_directories
from utils.eda_logger import setup_eda_logger
 


# Section 1:Configuration et initialisation

In [2]:
# Chargement de la configuration principale (Hydra)
cfg = load_config()

# Initialisation du logger
setup_eda_logger(cfg)
logger = logging.getLogger(__name__)

# Création des dossiers nécessaires à l'exécution (raw, interim, processed, reports, etc.)
create_directories(cfg)


2025-12-31 00:16:22,374 - utils.config_loader - INFO - Répertoire prêt : C:\Users\HP\Desktop\temp\TODO\SEMESTRE_1\ML1\ML-prediction-CO2\data\raw
2025-12-31 00:16:22,374 - utils.config_loader - INFO - Répertoire prêt : C:\Users\HP\Desktop\temp\TODO\SEMESTRE_1\ML1\ML-prediction-CO2\data\interim
2025-12-31 00:16:22,374 - utils.config_loader - INFO - Répertoire prêt : C:\Users\HP\Desktop\temp\TODO\SEMESTRE_1\ML1\ML-prediction-CO2\data\processed
2025-12-31 00:16:22,374 - utils.config_loader - INFO - Répertoire prêt : C:\Users\HP\Desktop\temp\TODO\SEMESTRE_1\ML1\ML-prediction-CO2\figures
2025-12-31 00:16:22,374 - utils.config_loader - INFO - Répertoire prêt : C:\Users\HP\Desktop\temp\TODO\SEMESTRE_1\ML1\ML-prediction-CO2\reports


# Section 2 : Chargement et inspection structurelle

Cette section se concentre sur le chargement initial des données à partir du fichier CSV et une inspection structurelle de base. L'objectif est d'obtenir une première vue d'ensemble du dataset, de vérifier son intégrité et d'identifier tout problème évident au niveau du format ou de la structure.

## Importation du CSV

In [3]:
# Chargement des données brutes
df_raw = load_data_raw(cfg)

# Validation dimensions
n_rows, n_cols = df_raw.shape
logger.info(f"Dataset chargé ({n_rows} lignes, {n_cols} colonnes)")



2025-12-31 00:16:22,429 - data.load_data - INFO - DataFrame chargé : 3376 lignes, 46 colonnes
2025-12-31 00:16:22,448 - data.load_data - INFO -  Intégrité des données validée (Aucune modification détectée).
2025-12-31 00:16:22,448 - __main__ - INFO - Dataset chargé (3376 lignes, 46 colonnes)


---
Le chargement nous donne un DataFrame avec 3376 lignes et 46 colonnes.

## Affichage pour première impression visuelle

In [4]:
# Forcer pandas à afficher toutes les colonnes
pd.set_option("display.max_columns", None)

# Forcer pandas à afficher toutes les lignes (si besoin)
pd.set_option("display.max_rows", None)

# Forcer pandas à afficher toute la largeur (évite les "...")
pd.set_option("display.width", None)



In [5]:
# Affichage des premières lignes
print("Premières 5 lignes :")
display(df_raw.head(5))

# Affichage des dernières lignes
print("Dernières 5 lignes :")
display(df_raw.tail(5))

# Affichage d'un échantillon aléatoire
print("Échantillon aléatoire de 20 lignes :")
display(df_raw.sample(20))

Premières 5 lignes :


OSEBuildingID  DataYear    BuildingType PrimaryPropertyType  \
0              1      2016  NonResidential               Hotel   
1              2      2016  NonResidential               Hotel   
2              3      2016  NonResidential               Hotel   
3              5      2016  NonResidential               Hotel   
4              8      2016  NonResidential               Hotel   

                  PropertyName          Address     City State  ZipCode  \
0         Mayflower park hotel    405 Olive way  Seattle    WA  98101.0   
1              Paramount Hotel  724 Pine street  Seattle    WA  98101.0   
2      5673-The Westin Seattle  1900 5th Avenue  Seattle    WA  98101.0   
3                    HOTEL MAX   620 STEWART ST  Seattle    WA  98101.0   
4  WARWICK SEATTLE HOTEL (ID8)    401 LENORA ST  Seattle    WA  98121.0   

  TaxParcelIdentificationNumber  CouncilDistrictCode Neighborhood  Latitude  \
0                    0659000030                    7     DOWNTOWN  47.61220   
1                    0659000220                    7     DOWNTOWN  47.61317   
2                    0659000475                    7     DOWNTOWN  47.61393   
3                    0659000640                    7     DOWNTOWN  47.61412   
4                    0659000970                    7     DOWNTOWN  47.61375   

   Longitude  YearBuilt  NumberofBuildings  NumberofFloors  PropertyGFATotal  \
0 -122.33799       1927                1.0              12             88434   
1 -122.33393       1996                1.0              11            103566   
2 -122.33810       1969                1.0              41            956110   
3 -122.33664       1926                1.0              10             61320   
4 -122.34047       1980                1.0              18            175580   

   PropertyGFAParking  PropertyGFABuilding(s)      ListOfAllPropertyUseTypes  \
0                   0                   88434                          Hotel   
1               15064                   88502     Hotel, Parking, Restaurant   
2              196718                  759392                          Hotel   
3                   0                   61320                          Hotel   
4               62000                  113580  Hotel, Parking, Swimming Pool   

  LargestPropertyUseType  LargestPropertyUseTypeGFA  \
0                  Hotel                    88434.0   
1                  Hotel                    83880.0   
2                  Hotel                   756493.0   
3                  Hotel                    61320.0   
4                  Hotel                   123445.0   

  SecondLargestPropertyUseType  SecondLargestPropertyUseTypeGFA  \
0                          NaN                              NaN   
1                      Parking                          15064.0   
2                          NaN                              NaN   
3                          NaN                              NaN   
4                      Parking                          68009.0   

  ThirdLargestPropertyUseType  ThirdLargestPropertyUseTypeGFA  \
0                         NaN                             NaN   
1                  Restaurant                          4622.0   
2                         NaN                             NaN   
3                         NaN                             NaN   
4               Swimming Pool                             0.0   

  YearsENERGYSTARCertified  ENERGYSTARScore  SiteEUI(kBtu/sf)  \
0                      NaN             60.0         81.699997   
1                      NaN             61.0         94.800003   
2                      NaN             43.0         96.000000   
3                      NaN             56.0        110.800003   
4                      NaN             75.0        114.800003   

   SiteEUIWN(kBtu/sf)  SourceEUI(kBtu/sf)  SourceEUIWN(kBtu/sf)  \
0           84.300003          182.500000            189.000000   
1           97.900002          176.100006            179.399994   
2

Dernières 5 lignes :


OSEBuildingID  DataYear        BuildingType PrimaryPropertyType  \
3371          50222      2016  Nonresidential COS              Office   
3372          50223      2016  Nonresidential COS               Other   
3373          50224      2016  Nonresidential COS               Other   
3374          50225      2016  Nonresidential COS  Mixed Use Property   
3375          50226      2016  Nonresidential COS  Mixed Use Property   

                             PropertyName           Address     City State  \
3371                Horticulture building  1600 S Dakota St  Seattle    WA   
3372  International district/Chinatown CC     719 8th Ave S  Seattle    WA   
3373                      Queen Anne Pool    1920 1st Ave W  Seattle    WA   
3374          South Park Community Center    8319 8th Ave S  Seattle    WA   
3375          Van Asselt Community Center  2820 S Myrtle St  Seattle    WA   

      ZipCode TaxParcelIdentificationNumber  CouncilDistrictCode  \
3371      NaN                    1624049080                    2   
3372      NaN                    3558300000                    2   
3373      NaN                    1794501150                    7   
3374      NaN                    7883603155                    1   
3375      NaN                    7857002030                    2   

               Neighborhood  Latitude  Longitude  YearBuilt  \
3371       GREATER DUWAMISH  47.56722 -122.31154       1990   
3372               DOWNTOWN  47.59625 -122.32283       2004   
3373  MAGNOLIA / QUEEN ANNE  47.63644 -122.35784       1974   
3374       GREATER DUWAMISH  47.52832 -122.32431       1989   
3375       GREATER DUWAMISH  47.53939 -122.29536       1938   

      NumberofBuildings  NumberofFloors  PropertyGFATotal  PropertyGFAParking  \
3371                1.0               1             12294                   0   
3372                1.0               1             16000                   0   
3373                1.0               1             13157                   0   
3374                1.0               1             14101                   0   
3375                1.0               1             18258                   0   

      PropertyGFABuilding(s)  \
3371                   12294   
3372                   16000   
3373                   13157   
3374                   14101   
3375                   18258   

                              ListOfAllPropertyUseTypes  \
3371                                             Office   
3372                                 Other - Recreation   
3373  Fitness Center/Health Club/Gym, Other - Recrea...   
3374  Fitness Center/Health Club/Gym, Food Service, ...   
3375  Fitness Center/Health Club/Gym, Food Service, ...   

     LargestPropertyUseType  LargestPropertyUseTypeGFA  \
3371                 Office                    12294.0   
3372     Other - Recreation                    16000.0   
3373     Other - Recreation                     7583.0   
3374     Other - Recreation                     6601.0   
3375     Other - Recreation                     8271.0   

        SecondLargestPropertyUseType  SecondLargestPropertyUseTypeGFA  \
3371                             NaN                              NaN   
3372                             NaN                              NaN   
3373  Fitness Center/Health Club/Gym                           5574.0   
3374  Fitness Center/Health Club/Gym                           6501.0   
3375  Fitness Center/Health Club/Gym                           8000.0   

     ThirdLargestPropertyUseType  ThirdLargestPropertyUseTypeGFA  \
3371                         NaN                             NaN   
3372                         NaN                             NaN   
3373               Swimming Pool                             0.0   
3374          Pre-school/Daycare                           484.0   
3375          Pre-school/Daycare                          1108.0   

     YearsENERGYSTARCertified  ENERGYSTARScore  SiteEUI(kBtu/sf)  \
3371      

Échantillon aléatoire de 20 lignes :


OSEBuildingID  DataYear          BuildingType  \
2694          26653      2016  Multifamily LR (1-4)   
2407          25546      2016        NonResidential   
1154          21115      2016        NonResidential   
2130          24556      2016  Multifamily MR (5-9)   
3315          50029      2016  Multifamily HR (10+)   
2805          27020      2016        NonResidential   
1837          23560      2016  Multifamily LR (1-4)   
3337          50062      2016        NonResidential   
3191          49736      2016  Multifamily LR (1-4)   
2873          27295      2016  Multifamily LR (1-4)   
1743          23293      2016  Multifamily LR (1-4)   
1691          23115      2016        NonResidential   
2071          24354      2016        NonResidential   
2296          25170      2016  Multifamily LR (1-4)   
412             571      2016        NonResidential   
2120          24532      2016        NonResidential   
3006          27845      2016  Multifamily HR (10+)   
2014          24150      2016  Multifamily LR (1-4)   
2700          26671      2016  Multifamily LR (1-4)   
3351          50088      2016  Multifamily LR (1-4)   

              PrimaryPropertyType                               PropertyName  \
2694         Low-Rise Multifamily                 University Park Apartments   
2407           Mixed Use Property  Compass Housing Alliance - Pioneer Square   
1154  Small- and Mid-Sized Office               Administration - Cassel Crag   
2130         Mid-Rise Multifamily                     Pontedera Condominiums   
3315        High-Rise Multifamily                               Walton Lofts   
2805               Residence Hall               Alpha Delta Pi Chapter House   
1837         Low-Rise Multifamily                     Tramonti at Lake Union   
3337                        Hotel                        Olympic Hospitality   
3191         Low-Rise Multifamily                      Cottage Grove Commons   
2873         Low-Rise Multifamily                              Jackson Manor   
1743         Low-Rise Multifamily                         Madison Apartments   
1691          Distribution Center                       Intermountain Supply   
2071                    Warehouse                      1760 4th Avenue South   
2296         Low-Rise Multifamily                          Cirque Apartments   
412           Distribution Center                Northwest Center Industries   
2120                    Warehouse          Kenyon Industrial Park Building B   
3006        High-Rise Multifamily                    (ID27845) COUNCIL HOUSE   
2014         Low-Rise Multifamily                     Seward Park Apartments   
2700         Low-Rise Multifamily                          North Towne Manor   
3351         Low-Rise Multifamily                            Vibe Apartments   

                                      Address     City State  ZipCode  \
2694                         5240 15th Ave NE  Seattle    WA  98105.0   
2407  210 Alaskan Way S / 77 S. Washington St  Seattle    WA  98104.0   
1154                           1218 Terry Ave  Seattle    WA  98101.0   
2130                 827 Hiawatha Place South  Seattle    WA  98144.0   
3315                               75 Vine St  Seattle    WA  98121.0   
2805                             1805 NE 47th  Seattle    WA  98177.0   
1837                  2920 Eastlake Ave. East  Seattle    WA  98102.0   
3337                        3926 Aurora Ave N  Seattle    WA  98103.0   
3191                     5444 Delridge Way SW  Seattle    WA  98106.0   
2873                         1526 NE 143rd ST  Seattle    WA  98125.0   
1743                          823 Madison St.  Seattle    WA  98102.0   
1691                       7901 1st Ave South  Seattle    WA  98108.0   
2071                    1760 4th Avenue South  Seattle    WA  98134.0   
2296                       621 Harvard Ave E.  Seattle    WA  98102.0   
412                   7272 W. Marginal Way S.  Seattle    WA  98108.0   
2

---
La colonne 'DataYear' est constante, ce qui est attendu puisque le dataset porte sur une seule campagne annuelle. Cette variable n’apporte donc pas de variance et pourra être ignorée dans la modélisation.

On observe une présence importante de valeurs manquantes dans les colonnes liées aux usages secondaires et tertiaires des bâtiments (SecondLargestPropertyUseType, ThirdLargestPropertyUseType, YearsENERGYSTARCertified, Comments). Cela traduit soit une absence d’information, soit des cas où le bâtiment n’a qu’un usage principal. Ces colonnes nécessiteront un exament approfondi.

Les colonnes lié à l'energie  montrent de nombreux zéros :  
- *PropertyGFAParking* est souvent à 0, ce qui est logique pour les bâtiments sans parking.  
- *SteamUse(kBtu)* et *NaturalGas(kBtu)* présentent des 0.00, ce qui peut indiquer une absence réelle de consommation ou une donnée manquante codée comme zéro. 
> Il faudra distinguer les deux cas pour éviter de biaiser les modèles. 


Electricity(kWh),la quantité annuelle d'éléctricité consommée présente d'énormes differences (ordre de grandeur),selon les batiments.

La colonne **ComplianceStatus** ( **Si une propriété a satisfait aux exigences d'analyse comparative énergétique pour l'année de déclaration en cours**), est presque toujours “Compliant”, ce qui suggère que la majorité des bâtiments respectent la réglementation. 

La colonne **Outlier** est vide avec des NaN, ce qui peut indiquer que les anomalies sont rares ou non encore identifiées.


## Informations structurelles

In [10]:
df_raw.info(verbose=True)

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 3376 entries, 0 to 3375
Data columns (total 46 columns):
 #   Column                           Non-Null Count  Dtype  
---  ------                           --------------  -----  
 0   OSEBuildingID                    3376 non-null   int64  
 1   DataYear                         3376 non-null   int64  
 2   BuildingType                     3376 non-null   object 
 3   PrimaryPropertyType              3376 non-null   object 
 4   PropertyName                     3376 non-null   object 
 5   Address                          3376 non-null   object 
 6   City                             3376 non-null   object 
 7   State                            3376 non-null   object 
 8   ZipCode                          3360 non-null   float64
 9   TaxParcelIdentificationNumber    3376 non-null   object 
 10  CouncilDistrictCode              3376 non-null   int64  
 11  Neighborhood                     3376 non-null   object 
 12  Latitude            

In [9]:
# Liste des colonnes
print("Colonnes :")
print(df_raw.columns.tolist())

Colonnes :
['OSEBuildingID', 'DataYear', 'BuildingType', 'PrimaryPropertyType', 'PropertyName', 'Address', 'City', 'State', 'ZipCode', 'TaxParcelIdentificationNumber', 'CouncilDistrictCode', 'Neighborhood', 'Latitude', 'Longitude', 'YearBuilt', 'NumberofBuildings', 'NumberofFloors', 'PropertyGFATotal', 'PropertyGFAParking', 'PropertyGFABuilding(s)', 'ListOfAllPropertyUseTypes', 'LargestPropertyUseType', 'LargestPropertyUseTypeGFA', 'SecondLargestPropertyUseType', 'SecondLargestPropertyUseTypeGFA', 'ThirdLargestPropertyUseType', 'ThirdLargestPropertyUseTypeGFA', 'YearsENERGYSTARCertified', 'ENERGYSTARScore', 'SiteEUI(kBtu/sf)', 'SiteEUIWN(kBtu/sf)', 'SourceEUI(kBtu/sf)', 'SourceEUIWN(kBtu/sf)', 'SiteEnergyUse(kBtu)', 'SiteEnergyUseWN(kBtu)', 'SteamUse(kBtu)', 'Electricity(kWh)', 'Electricity(kBtu)', 'NaturalGas(therms)', 'NaturalGas(kBtu)', 'DefaultData', 'Comments', 'ComplianceStatus', 'Outlier', 'TotalGHGEmissions', 'GHGEmissionsIntensity']


Voir le dictionnaire des donnée dans raports/ pour des définitions

On constate que les types des variables sont coherent.(un int déguisé en float ne pose pas de probléme)

>La variable zipcode est identifié comme int mais dois être traité comme une variable qualitatif

> colonnes :46 au total, couvrant identifiants, localisation, caractéristiques, usages, performances, consommations, émissions

>le 'WN' signifie Weather Normalized, une normalisation importante pour comparer les années

Nous avons aussi quelques variables atypiques :

> **DefaultData** booléen pour usage de données par défaut
>**Outlier** Si une propriété est une valeur aberrante élevée ou faible (O/N)

Cela necessite une documentation plus approfondi

# Section 3 : Typologie des variables

Dans cette section, nous présentons le dictionnaire des données. Afin de garantir la cohérence et la fidélité aux sources, nous avons repris la documentation officielle et en avons traduit automatiquement les éléments essentiels. Les informations pertinentes ont été extraites et adaptées pour ce projet, de manière à offrir une lecture directe et exploitable.

Le document complet est disponible dans le dossier  reports/notebooks/data_dictionary_auto.pdf. Il constitue la référence officielle pour la description des variables et sera utilisé comme support de documentation dans le cadre du projet.

## Structure et cohérence des variables

On remarque que le dictionnaire est très riche et couvre à la fois des identifiants administratifs (OSEBuildingID, TaxParcelIdentificationNumber), des informations géographiques (adresse, latitude, longitude, quartier), et des caractéristiques physiques des bâtiments (année de construction, nombre d’étages, surface totale). Cette structuration permet de relier chaque enregistrement à une entité réelle et de contextualiser les mesures énergétiques.
Un point intéressant est la présence de variables redondantes ou complémentaires, comme PropertyGFATotal, PropertyGFAParking et PropertyGFABuilding(s).

## Données énergétiques et enjeux métier


SiteEUI, SourceEUI, consommation d’électricité, gaz, vapeur, émissions de GES, Ces variables traduisent directement l’objectif du benchmarking : comparer les bâtiments entre eux et identifier les leviers de réduction.
On note aussi la présence de champs qualitatifs comme ComplianceStatus ou Outlier, qui ajoutent une dimension réglementaire et statistique. Cela montre que le dataset n’est pas seulement descriptif, mais qu’il sert aussi à évaluer la conformité et détecter les anomalies.
Enfin, certaines colonnes sont très incomplètes (YearsENERGYSTARCertified, Comments, Outlier), ce qui rappelle une réalité humaine : la difficulté de collecter des données exhaustives. C’est une limite mais aussi une information précieuse sur la fiabilité et la représentativité des analyses.

## Classification des variables


## 1. Identification et localisation

Ces variables servent à identifier le bâtiment de manière unique et à le situer géographiquement.

| Variable | Type | Description |
| --- | --- | --- |
| **OSEBuildingID** | Numérique (ID) | Identifiant unique de la propriété. |
| **PropertyName** | Texte | Nom officiel du bâtiment. |
| **TaxParcelIdentificationNumber** | Texte | Identifiant fiscal de la parcelle (King County). |
| **Address** | Texte | Adresse physique. |
| **City** | Texte | Ville (Seattle). |
| **State** | Texte | État (WA). |
| **ZipCode** | Catégoriel | Code postal. |
| **CouncilDistrictCode** | Catégoriel | District municipal (1 à 7). |
| **Neighborhood** | Catégoriel | Zone de quartier définie par la ville. |
| **Latitude** | Numérique | Coordonnée géographique nord-sud. |
| **Longitude** | Numérique | Coordonnée géographique est-ouest. |

---

## 2. Caractéristiques morphologiques et temporelles

Variables décrivant la structure physique et l'âge du bâtiment.

| Variable | Type | Description |
| --- | --- | --- |
| **DataYear** | Temporel | Année de l'enregistrement (2016). |
| **YearBuilt** | Temporel | Année de construction ou de rénovation majeure. |
| **NumberofBuildings** | Numérique | Nombre de bâtiments sur le site. |
| **NumberofFloors** | Numérique | Nombre d'étages du bâtiment. |
| **PropertyGFATotal** | Numérique | Surface totale (Bâtiment + Parking) en pieds carrés. |
| **PropertyGFABuilding(s)** | Numérique | Surface brute hors parking. |
| **PropertyGFAParking** | Numérique | Surface totale des parkings. |

---

## 3. Typologie et usage des sols

Classification du bâtiment selon son activité principale et secondaire.

| Variable | Type | Description |
| --- | --- | --- |
| **BuildingType** | Catégoriel | Classification globale (ex: NonResidential, Campus). |
| **PrimaryPropertyType** | Catégoriel | Usage principal (>50%) selon Portfolio Manager. |
| **ListOfAllPropertyUseTypes** | Texte | Liste exhaustive de tous les usages sur le site. |
| **LargestPropertyUseType** | Catégoriel | Usage occupant la plus grande surface. |
| **LargestPropertyUseTypeGFA** | Numérique | Surface associée à l'usage le plus important. |
| **SecondLargestPropertyUseType** | Catégoriel | Deuxième usage par importance de surface. |
| **SecondLargestPropertyUseTypeGFA** | Numérique | Surface associée au deuxième usage. |
| **ThirdLargestPropertyUseType** | Catégoriel | Troisième usage par importance de surface. |
| **ThirdLargestPropertyUseTypeGFA** | Numérique | Surface associée au troisième usage. |

---

## 4. Performance et Certification

Indicateurs d'efficacité énergétique et labels environnementaux.

| Variable | Type | Description |
| --- | --- | --- |
| **ENERGYSTARScore** | Numérique (0-100) | Score de performance relative (EPA). |
| **YearsENERGYSTARCertified** | Texte | Historique des années de certification Energy Star. |

---

## 5. Intensité d'usage de l'Énergie (EUI)

Ratios de consommation par unité de surface, normalisés ou non (Indicateurs clés de performance).

| Variable | Type | Unité | Description |
| --- | --- | --- | --- |
| **SiteEUI(kBtu/sf)** | Numérique | kBtu/sq.ft | Énergie consommée sur site par pied carré. |
| **SiteEUIWN(kBtu/sf)** | Numérique | kBtu/sq.ft | EUI du site normalisé selon la météo. |
| **SourceEUI(kBtu/sf)** | Numérique | kBtu/sq.ft | Énergie à la source (inclut pertes réseau) par pied carré. |
| **SourceEUIWN(kBtu/sf)** | Numérique | kBtu/sq.ft | Source EUI normalisé selon la météo. |

---

## 6. Consommation Énergétique par source

Volumes de consommation bruts selon le type d'énergie.

| Variable | Type | Unité | Description |
| --- | --- | --- | --- |
| **SiteEnergyUse(kBtu)** | Numérique | kBtu | Consommation totale annuelle du site. |
| **SiteEnergyUseWN(kBtu)** | Numérique | kBtu | Consommation totale normalisée (météo). |
| **Electricity(kWh)** | Numérique | kWh | Consommation électrique brute. |
| **Electricity(kBtu)** | Numérique | kBtu | Consommation électrique convertie en kBtu. |
| **NaturalGas(therms)** | Numérique | therms | Consommation de gaz naturel brute. |
| **NaturalGas(kBtu)** | Numérique | kBtu | Consommation de gaz naturel convertie en kBtu. |
| **SteamUse(kBtu)** | Numérique | kBtu | Consommation de vapeur urbaine. |

---

## 7. Émissions de gaz à effet de serre 

Mesure de l'impact environnemental du bâtiment.

| Variable | Type | Unité | Description |
| --- | --- | --- | --- |
| **TotalGHGEmissions** | Numérique | Tonnes CO2e | Émissions totales de gaz à effet de serre. |
| **GHGEmissionsIntensity** | Numérique | kg CO2e/sq.ft | Émissions par unité de surface. |

---

## 8. qualité des données et conformité

Variables indiquant la fiabilité des informations et l'état réglementaire.

| Variable | Type | Description |
| --- | --- | --- |
| **DefaultData** | Booléen | Indique si des valeurs par défaut ont été utilisées. |
| **ComplianceStatus** | Catégoriel | Statut vis-à-vis de l'ordonnance de benchmarking. |
| **Outlier** | Catégoriel | Indique si la donnée est une valeur aberrante. |
| **Comments** | Texte | Remarques textuelles additionnelles (très peu rempli). |

---